sdks/python/apache_beam/examples/inference/data/sentences.txt

```diff
@@ -0,0 +1,10 @@
+The capital of France is Paris .
```

Comment:
I think we need to add Apache License here

Reply:
We could host it in GCS or pass via `beam.Create([list_of_strings])` if input not specified.